src/components/Footer.tsx

```diff
@@ -0,0 +1,56 @@
+export default function Footer() {
+  return (
+    <footer className="bg-transparent position:relative flex flex-col justify-center items-center w-full before:content-[''] before:absolute before:w-full before:h-[14.7rem] before:bg-[url('/waves.svg')] before:bg-no-repeat before:bg-top before:-z-10">
+      
+
+
+      <img src = "logoGrace.svg" alt=" "/>
+
+      <ul className="flex ">
+        <a href="https://www.instagram.com/graceusp/?hl=pt-br">
+          <svg
+            id="botaoInsta"
```

Comment:
**:hammer: Ajuste**

Ao invés de importar o ícone svg no código, podemos usar a biblioteca "react-icons". Ela possui várias bibliotecas de ícone e sugiro a gente utilizar a biblioteca "Feather" para padronizar todos os ícones

https://react-icons.github.io/react-icons/icons/fi/

Nessa página, você pode procurar os ícones dentro dessa biblioteca (ela possui os ícones de instagram e e-mail)

Para usar, basta vc clicar no ícone q vc quer importar. Aí você vai colocar o import dele no ínicio do seu código e chamar como se fosse um componente

![image](https://github.com/petsi-each/Grace-Website/assets/62656745/92a02e5b-291b-4120-a7bb-e3e14700bd8a)


Para mudar o tamanho, você adicionar a propriedade de "size". Ex:

`<FiX size={30} />`

Para mudar a cor, você cria uma tag que engloba o ícone e dá uma propriedade de text (como se fosse um texto)
```html
<i className="text-brancoGrace">
<FiX size={30} />
</i>
```

Ps: sim, a tag `<i>` é para itálico, mas também é utilizada para ícones (para saber mais: https://www.w3schools.com/icons/)